tests/framework/extensions/etcdsnapshot/config.go

```diff
@@ -5,6 +5,11 @@ const (
 )
 
 type Config struct {
-	UpgradeKubernetesVersion string `json:"upgradeKubernetesVersion" yaml:"upgradeKubernetesVersion"`
-	SnapshotRestore          string `json:"snapshotRestore" yaml:"snapshotRestore"`
+	UpgradeKubernetesVersion     string `json:"upgradeKubernetesVersion" yaml:"upgradeKubernetesVersion"`
+	SnapshotRestore              string `json:"snapshotRestore" yaml:"snapshotRestore"`
+	ControlPlaneConcurrencyValue string `json:"controlPlaneConcurrencyValue" yaml:"controlPlaneConcurrencyValue"`
+	ControlPlaneUnavailableValue string `json:"controlPlaneUnavailableValue" yaml:"controlPlaneUnavailableValue"`
+	WorkerConcurrencyValue       string `json:"workerConcurrencyValue" yaml:"workerConcurrencyValue"`
+	WorkerUnavailableValue       string `json:"workerUnavailableValue" yaml:"workerUnavailableValue"`
+	SnapshotCounter              int    `json:"snapshotCounter" yaml:"snapshotCounter" default:"1"`
```

Comment:
Why don't other vars need a default value too? I feel that this should be set manually in static tests, and omitted for dynamic tests. What do you think?

Also, this should have a more clear name. Maybe `RecurringRestores`?

Reply:
I only kept a default for `SnapshotCounter` to ensure that we don't run multiple times and only once. I can rename that no problem as well as set default values. I will make that change.